tests/Integration/Http/Controller/ForgotControllerTest.php

```diff
@@ -51,26 +51,30 @@ public function indexDisplaysCorrectForm()
     public function sendResetDisplaysCorrectMessage()
     {
         $accounts = m::mock(AccountManagement::class);
-        $accounts->shouldReceive('findByLogin')->andReturn($this->createUser());
+        $accounts->shouldReceive('findByLogin')
+            ->withArgs(['someone@example.com'])
```

Comment:
👍 

I believe quite a few tests where we currently set up expectations could benefit from being more specific.